Recaptcha/RecaptchaVerifier.php

```diff
@@ -39,11 +39,14 @@ public function __construct(ReCaptcha $reCaptcha, RequestStack $requestStack, $e
 
     /**
      * Verify reCaptcha response.
+     *
+     * @param $recaptchaValue
+     *
+     * @throws RecaptchaException
      */
-    public function verify()
+    public function verify($recaptchaValue)
```

Comment:
I still don't see how `$recaptchaValue` is passed to `verify`. The `Recaptcha2Validator` is still calling `verify` without any argument.